src/components/moderation/ReportDialog/utils/useReportOptions.ts

```diff
@@ -1,121 +1,321 @@
+        {
+          title: _(msg`Hate Speech`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentHateSpeech',
+        },
+        {
+          title: _(msg`Doxxing`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentDoxxing',
+        },
+        {
+          title: _(msg`Other Harassing or Hateful Content`),
+          reason: 'tools.ozone.report.defs#reasonHarassmentOther',
+        },
+      ],
+    },
+    misleading: {
+      key: 'misleading',
+      title: _(msg`Misleading`),
+      description: _(msg`Spam, scams, false info or impersonation`),
+      sort: 5,
+      options: [
+        {
+          title: _(msg`Fake Account or Bot`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingBot',
+        },
+        {
+          title: _(msg`Impersonation`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingImpersonation',
+        },
         {
-          reason: ComAtprotoModerationDefs.REASONSPAM,
           title: _(msg`Spam`),
-          description: _(msg`Excessive or unwanted messages`),
+          reason: 'com.atproto.moderation.defs#reasonSpam',
+        },
+        {
+          title: _(msg`Scam`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingScam',
+        },
+        {
+          title: _(msg`Unlabelled GenAI or Synthetic Content`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingSyntheticContent',
+        },
+        {
+          title: _(msg`Harmful False Claims`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingMisinformation',
         },
         {
-          reason: ComAtprotoModerationDefs.REASONSEXUAL,
-          title: _(msg`Unwanted Sexual Content`),
-          description: _(msg`Inappropriate messages or explicit links`),
+          title: _(msg`Other Misleading Content`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingOther',
         },
-        ...common,
       ],
-      list: [
+    },
+    ruleBreaking: {
+      key: 'ruleBreaking',
+      title: _(msg`Breaking Network Rules`),
+      description: _(msg`Hacking, stolen content or prohibited sales`),
+      sort: 6,
+      options: [
         {
-          reason: ComAtprotoModerationDefs.REASONVIOLATION,
-          title: _(msg`Name or Description Violates Community Standards`),
-          description: _(msg`Terms used violate community standards`),
+          title: _(msg`Hacking or System Attacks`),
+          reason: 'tools.ozone.report.defs#reasonRuleSiteSecurity',
+        },
+        {
+          title: _(msg`Stolen Content`),
+          reason: 'tools.ozone.report.defs#reasonRuleStolenContent',
+        },
+        {
+          title: _(msg`Promoting or Selling Prohibited Items of Services`),
+          reason: 'tools.ozone.report.defs#reasonRuleProhibitedSales',
+        },
+        {
+          title: _(msg`Banned User Returning`),
+          reason: 'tools.ozone.report.defs#reasonRuleBanEvasion',
+        },
+        {
+          title: _(msg`Other`),
```

Comment:
```suggestion
          title: _(msg`Other Network Rule-breaking`),
```
nit: Suggest expanding this rather than simply `Other`